src/scf/mixing.jl

```diff
@@ -6,8 +6,13 @@
 
 # The interface is `mix(m, basis, ρin, ρout) -> ρnext`
 
-"""
-Kerker mixing: J^-1 ≈ α*G^2/(G0^2 + G^2)
+using LinearMaps
+using IterativeSolvers
+
+@doc raw"""
+Kerker mixing: ``J^{-1} ≈ \frac{α G^2}{(G0^2 + G^2}``
+
+Abinit calls ``1/G0`` the dielectric screening length (parameter *dielng*)
```

Comment:
We should not use this terminology : dielectric usually refers to insulators and semiconductors. Rather, I'd say that 1/G0 is the Thomas Fermi screening length

Reply:
Ok, I made it clear this is just a note. This helps to relate the parameters to what other codes are doing. I'd like to keep it at least mentioned.